lib/form_data.js

```diff
@@ -349,10 +352,12 @@ FormData.prototype.getLength = function(cb) {
 };
 
 FormData.prototype.submit = function(params, cb) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/form-data/form-data/file/2429836185/issues/source?bid=2805153&fileBranchId=3175127#l354)